packages/purpose-outbound-writer/src/config/config.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  PurposeTopicConfig,
+  KafkaConsumerConfig,
+} from "pagopa-interop-commons";
+import { logLevel } from "kafkajs";
+import { z } from "zod";
+
+const PurposeOutboundWriterConfig = KafkaConsumerConfig.and(PurposeTopicConfig)
+  .and(
+    z.object({
+      PURPOSE_OUTBOUND_TOPIC: z.string(),
+      PRODUCER_KAFKA_BROKERS: z.string(),
+      PRODUCER_KAFKA_CLIENT_ID: z.string(),
+      PRODUCER_KAFKA_DISABLE_AWS_IAM_AUTH: z.literal("true").optional(),
+      PRODUCER_KAFKA_LOG_LEVEL: z
+        .enum(["NOTHING", "ERROR", "WARN", "INFO", "DEBUG"])
+        .default("WARN"),
+      PRODUCER_KAFKA_REAUTHENTICATION_THRESHOLD: z
+        .number()
+        .default(20)
+        .transform((n) => n * 1000),
```

Comment:
these configs are already in `KafkaConfig`, any reason why they have been added here?

Reply:
This script needs two `KafkaConfig`, one for the producer (which will produce outbound events) and one for the consumer (which will consume interop internal events). 

The config for the consumer is taken from the `KafkaConsumerConfig` zod schema (which is a superset of the `KafkaConfig`), while the config for the producer is the same of the `KafkaConfig` but in order to differentiate it from the consumer's I copied and pasted all his property and prefixed all with `PRODUCER_`. 

Maybe we should create `KafkaProducerConfig` in the commons, what do you think?

Reply:
you're right, I totally missed that.
Adding the config in commons may be worth, at least to guarantee that all producers use the same name for variables.
I don't see now services producing on multiple topics, but in case we can use a specific strategy (or refactor a bit)